src/protocol/ws/non_wasm.rs

```diff
@@ -50,57 +51,89 @@ use tokio_tungstenite::WebSocketStream;
 #[async_trait]
 impl Connection for Client {
     type Request = (i64, Message);
-    type Response = Value;
+    type Response = Result<DbResponse>;
     type Client = Self;
 
     fn new(method: Method) -> Self {
         Self { id: 0, method }
     }
 
-    async fn send<A>(
+    async fn send(
         &mut self,
         router: &Router<Self>,
-        msg: A,
-    ) -> Result<oneshot::Receiver<Result<Value>>>
-    where
-        A: Serialize + Send,
-    {
+        Param(first, second): Param,
```

Comment:
I'll be adding a `relate` method on the WebSocket RPC. This will have 3/4 arguments. I presume we'll change this to a `Param(one, two, three, four)`. Is this the best way of doing this?

```js
relate(from, kind, with, data);
```
`from`: a record id or param, or array
`kind`: a string (a table name)
`with`: a record id or param, or array
`data`: the fields/metadata for the edge record

Reply:
I will change this to `Vec<Value>` then. I was trying to avoid an allocation here until when we would actually need it.